### DISCUSSION TITLE
is it possible to enumerate multiple protocols?

### DISCUSSION THREAD

**Discussion by SinghNanak** [Pennyw0rth/NetExec#346] [Q&A] [answered]
I wanted to scan smb, wmi, winrm together at once but i couldn't without a script. I think it would be great to have that ability built-in. However, this is possible by scripting and users who want this functionality can develop a custom script according to there own need. what are you suggestions should we have this feature natively?

**Comment by Marshall-Hallenbeck** [comment, marked as answer]
I've thought about this in the past, but each protocol has different, and many times, unique parameters, so it would be a massive pain to sort through all of that.

What I do, and what I suggest everyone does, is scan each protocol separately, so your results don't get co-mingled from protocol to protocol.

Yes, that requires writing a bash script and probably a forloop, but it's quite easy :)
